packages/ui/src/app/pages/WorkingGroups/UpcomingOpening.tsx

```diff
@@ -74,10 +74,18 @@ export const UpcomingOpening = () => {
               </BadgeStatus>
             </BadgesRow>
             <Statistics>
-              <TokenValueStat title="Current budget" tooltipText="Lorem ipsum..." value={opening.budget} />
-              <DurationStatistics title="Opening Expected duration" value={opening.expectedEnding} />
+              <DurationStatistics title="Time to begun" value={opening.expectedStart} />
```

Comment:
```suggestion
              <DurationStatistics title="Time to begin" value={opening.expectedStart} />
```
Looks like a strange typo